src/manifest_store.rs

```diff
@@ -17,6 +17,18 @@ use crate::transactional_object_store::{
     DelegatingTransactionalObjectStore, TransactionalObjectStore,
 };
 
+/// read-only access to the latest manifest file
```

Comment:
I don't totally understand rust crate/packaging convention so I decided to just make this method public to avoid exposing any guts of this module (the return type is just `Option<String>` so we can change things underneath the hood without worrying about leaking anything to users of this method).

I'm open to suggestions about better ways to go about this, but doing it this way is nice because we don't need to open an entire database instance just to read the manifest file.

Reply:
I think the concern is more that this would require us to make `Manifest` and `CoreDbState` a part of the public API of slatedb. I'd rather not do this right now (if ever), and just return the serialized representation.

As an aside, I feel like this is out of place in this module. WDYT of adding an admin module that exposes what the cli needs publicly, and putting this function there?

Reply:
Yeah, @rodesai got my concern -- which is making `Manifest`, `CoreDbState` and everything downstream of that public API. I'm happy to move this to an admin module 👍